src/providers/details/tabs/Credentials/components/utils/validateInsecureSkipVerify.ts

```diff
@@ -0,0 +1,19 @@
+import { type ValidationMsg, ValidationState } from 'src/providers/utils/types';
+
+export const validateInsecureSkipVerify = (value: string): ValidationMsg => {
```

Comment:
1. all msgs are not translated
2. ['true', 'false', ''].includes(value) is valid? empty value is ok?